pika/adapters/base_connection.py

```diff
@@ -307,8 +306,6 @@ def _handle_ioloop_stop(self):
         """
         if self.stop_ioloop_on_close and self.ioloop:
             self.ioloop.stop()
-        elif self.WARN_ABOUT_IOLOOP:
-            LOGGER.warning('Connection is closed but not stopping IOLoop')
```

Comment:
Yes, the person who reported this issue mentions it specifically:

> I'll also point out that the "Connection is closed but not stopping IOLoop" is annoying; the IOLoop does not belong to pika, so it isn't pika's place to say whether the IOLoop should be running or stopped.

It could be changed to a debug message, but I don't see what value it adds since the message doesn't apply to other IO loop types.